src/js/collections/maps/AssetCategories.js

```diff
@@ -0,0 +1,57 @@
+"use strict";
+
+define([
+  "backbone",
+  "models/maps/AssetCategory",
+  "models/maps/Map",
+  "collections/maps/MapAssets",
+], function (
+  Backbone,
+  AssetCategory,
+  MapModel,
+  MapAssets,
+) {
+  /**
+   * @classdesc A AssetCategories collection is a group of AssetCategory models - models
+   * that provide the information required to render geo-spatial data in categories,
+   * including people, infrastructure, permafrost, etc.
+   * @classcategory Collections/Maps
+   * @class AssetCategories
+   * @extends Backbone.Collection
+  //  TODO: yvonneshi - update
+  //  * @since x.x.x
```

Comment:
Current release is 2.27.1 and this is targeting end of March release. How do I derive the release number here?

Reply:
Robyn fills in these x.x.x strings during release time. Safe to leave as x.x.x